PowerDocu.AppDocumenter/AppDocumentationContent.cs

```diff
@@ -24,7 +24,7 @@ class AppDocumentationContent
         public AppDocumentationContent(AppEntity app, string path)
         {
             NotificationHelper.SendNotification("Preparing documentation content for " + app.Name);
-            folderPath = path + CharsetHelper.GetSafeName(@"\AppDoc - " + app.Name + @"\");
+            folderPath = path + CharsetHelper.GetSafeName(@"\AppDoc " + app.Name + @"\");
```

Comment:
The '-' is read in DevOps Wiki's as a space, which got the wiki structure very confused so this had to be removed